astro/src/content/quickstarts/quickstart-javascript-nextjs-web.mdx

```diff
@@ -0,0 +1,283 @@
+
+Make sure once again that you are in the directory `complete-application/`.
+
+Copy the `.env.example` to `.env.local`
+
+```shell
+cp .env.example .env.local
+```
+
+The environment variables will match what is in the `/kickstart/kickstart.json` file that is used when first starting FusionAuth. If you have changed anything, like creating a new Application you will need to update these environment variables accordingly.
+
+`.env.local`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/.env.example"}
+  lang="ts" />
+
+Install required packages
+
+```shell
+npm install
+```
+
+### Walkthrough Example
+
+In this section, you'll set up a basic Next.js application with two pages. 
+
+1. Homepage
+1. Account - protected
+
+#### Setup Your Environment
+
+Create a new application using the `npx`.
+
+```shell
+npx create-next-app@latest changebank
+```
+
+Accept all of the default prompts like below
+
+![npx installation defaults](/img/articles/quickstart-javascript-nextjs-web/npx-create-next-app.png)
+
+Make sure you are in your new directory `changebank`
+
+```shell
+cd changebank
+```
+
+Install [NextAuth.js](https://next-auth.js.org/getting-started/example), like any tutorial I will show you what this magic does later.
+
+```shell
+npm install next-auth
+```
+
+Copy environment variables from our complete application example.
+
+```shell
+cp ../complete-application/.env.example .env.local
+```
+
+Also copy an image file into a new directory within `public` called `img`.
+
+```shell
+mkdir ./public/img && cp ../complete-application/public/img/money.jpg ./public/img/money.jpg
+```
+
+As we will be recreating all of the files in our app directory, please delete all files within `/src/app`.
+
+```shell
+rm -rf ./src/app && mkdir ./src/app
+```
+
+#### Authentication API
+
+Next.js 13.2 introduced <a href="https://nextjs.org/docs/app/building-your-application/routing/router-handlers" target="_blank">Route Handlers</a>, which are the preferred way to handle REST-like requests. In the `Changebank` application we can configure NextAuth.js FusionAuth's provider in a new route handler by creating a file within `src/app/api/auth/[...nextauth]/route.ts`.
+
+On first load of Next.js this file will make sure that we have all of the correct environment variables. The variables are then exported in an object called `authOptions` which can be imported on the server when we need to get our session using `getServerSession`.
+
+The `FusionAuthProvider` is then provided to `NextAuth` as a provider for any `GET` or `POST` commands that are sent to the `/api/auth/*` route.
+
+`src/app/api/auth/[...nextauth]/route.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/api/auth/[...nextauth]/route.ts"}
+  lang="ts" />
+
+#### Styles
+
+Create a new file `src/app/globals.css` copy the below CSS for the changebank application.
+
+`src/app/globals.css`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/globals.css"}
```

Comment:
I don't think this is important to the authentication part of the application.  I think this should be something that is already there and not take up space in the quickstart.  Something like the styles for the objects are defined in the src/app/globals.css file and you can update them there if you like.

I know you are trying to recreate the app from scratch, but we need to think a little more about how we want to present these quickstarts I think.  The more I am going through this, I think they should run out of the box and we just explain it vs recreating all from scratch.